kfs-core/src/main/java/edu/arizona/kfs/fp/document/DisbursementVoucherDocument.java

```diff
@@ -96,33 +99,29 @@ public void prepareForSave() {
             accountingLineExtension.setDocumentNumber(accountingLine.getDocumentNumber());
             accountingLineExtension.setSequenceNumber(accountingLine.getSequenceNumber());
         }
-        
-     // START KATTS-1961 Tag and JSP for DV, PREQ and CM Documents
+
         getIncomeTypeHandler().removeZeroValuedIncomeTypes();
-        // END KATTS-1961
-        
-        //KATTS-1939 Add New Search Fields to DV, PREQ and CM Documents 
-        // Only update paid year if the document is in final status 
-        if (KewApiConstants.ROUTE_HEADER_FINAL_CD.equals( getDocumentHeader().getWorkflowDocument().getStatus().getCode() ) ) {
-        	if (paidDate != null) {
-                setDvPaidYear(getPaidDate().toString().substring(0, 4));                
-            }
-            else {
-               setDvPaidYear(null);
+
+        // Only update paid year if the document is in final status
+        if (KewApiConstants.ROUTE_HEADER_FINAL_CD.equals(getDocumentHeader().getWorkflowDocument().getStatus().getCode())) {
+            if (paidDate != null) {
+                setDvPaidYear(getPaidDate().toString().substring(0, 4));
+            } else {
+                setDvPaidYear(null);
             }
-        }        
+        }
+
+        setDv1099Ind(false);
         for (DisbursementVoucherIncomeType incomeType : getIncomeTypes()) {
-            if ((StringUtils.isBlank(incomeType.getIncomeTypeCode()) || incomeType.getIncomeTypeCode().equals(KFSConstants.IncomeTypeConstants.INCOME_TYPE_NON_REPORTABLE_CODE))) {
-                setDv1099Ind(false);
-            }
-            else {
+            boolean isCodeExist = StringUtils.isNotBlank(incomeType.getIncomeTypeCode());
+            boolean isReportable = !incomeType.getIncomeTypeCode().equals(KFSConstants.IncomeTypeConstants.INCOME_TYPE_NON_REPORTABLE_CODE);
+            if (isCodeExist && isReportable) {
                 setDv1099Ind(true);
-                break;

```

Comment:
Based on discussions with the BAs, the size of getIncomeTypes() is usually 1, with the rare possibility of a few more. There was no compelling reason why to remove the break, other than readability. If you really want, I can put the break back in.